tests/transactional_system/transactional_update.pm

```diff
@@ -70,15 +70,21 @@ sub run {
     my $snap = script_output "snapper list | tail -1 | cut -d'|' -f$f | tr -d ' *'";
 
     record_info 'Update #1', 'Add repository and update - snapshot #2';
-    # Only CaaSP needs an additional repo for testing
-    assert_script_run 'zypper ar utt.repo' if is_caasp 'caasp';
-    trup_call 'cleanup up';
-    check_reboot_changes;
-    check_package 'up';
+    # Don't use tests requiring repos in staging
+    if (is_staging) {
```

Comment:
This is true only for Kubic. `is_staging` does not work on CaaSP (STAGING var is not defined) and I don't see reason to skip update tests there.

Reply:
well the fact that is_staging doesn't work on CaaSP means you aren't skipping the test.. so this has no impact for you..but..you probably should be skipping the tests
Else you're going to run the risk of packages coming from other repos which won't be compatible with the stuff currently in staging - that's how we hit this problem here - Kubic stagings were doing weird stuff like down/sidegrading kernels when testing stagings with new kernels ;)

Reply:
So far we have only one repo in caasp - for testing updates. There is no risk in downgrading / no difference to non-staging env.
You are right it has no inpact, I added is_opensuse just to make it obvious.